### PR TITLE
Adding new predefined metrics to symbolic module

### LINKED ISSUE #309: Add loads of predefined metrics in symbolic module
🐞 **What bugs you?**
  - Have predefined metrics in *symbolic* module

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by kauii8school**
Fixes #309
I've added in the barriola vilekin metric, berttoti kasner metric, and bessel gravitational wave metrics to the symbolic module. I should be adding a few more soon as well.

@ritzvik

### CODE REVIEW

**Comment by ritzvik** [comment]
Okay, einsteinpy follows strict formatting as described here : https://docs.einsteinpy.org/en/latest/dev_guide.html#code-linting

Also, you would need to add the entries to `symbolic/predefined/__init__.py` and add tests to `tests/test_symbolic/test_predefined/test_all.py`. Appropriate documentation would have to be added in `docs/source/api/symbolic/predefined/full_list.rst`.

File: src/einsteinpy/symbolic/predefined/bessel_gravitational_wave.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from einsteinpy.symbolic import constants
+from einsteinpy.symbolic.metric import MetricTensor
+from sympy import Function, Symbol, besselj, cos, diag, exp, sin, sqrt, symbols
+
+
+def Bessel_Gravitational_Wave(C=symbols("C")):
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/predefined/bessel_gravitational_wave.py]: Same:)

And please take care of all the instances below. Sorry for pointing out so teeny-tiny discrepancies but they are to maintain consistency.

Comment (kauii8school) [reply src/einsteinpy/symbolic/predefined/bessel_gravitational_wave.py]: No worries! I think it is important too, I just missed it haha

**Comment by ritzvik** [comment]
@kauii8school Thanks for the much needed PR. I am going to merge this. Cheers!

Also, I took the liberty to consolidate your commits. It would be good if you could try to do this from next time onwards.

**Comment by ritzvik** [comment]
One more thing.
Please try to create separate branches on git for addressing separate issues. Do not make PRs from `master`.

**Pull request merged**